src/services/ProblemEditorService.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+export class ProblemEditorService {
+
+    constructor() {
+        this.axios = axios.create({
+            baseURL: process.env.REACT_APP_EXAM_SVC_BASE_URL,
+            timeout: 10000
```

Comment:
Change baseURL to `process.env.REACT_APP_PROBLEM_SVC_BASE_URL`.
And add `REACT_APP_PROBLEM_SVC_BASE_URL=http://api.judgegirl.beta.pdlab.csie.ntu.edu.tw` in `.env` file.